Criptomining Hijack unminified.1blockpkg

```diff
@@ -0,0 +1,348 @@
+[
+  {
+    "id": "7337BF15-3EFA-43D0-BB18-B6A3E1A513AC",
+    "name": "Criptomining Hijack",
+    "rules": [
+      {
+        "content": {
+          "action": {
+            "type": "block"
+          },
+          "trigger": {
+            "url-filter-is-case-sensitive": true,
+            "url-filter": "https?://coin-?hive\\.com",
+            "load-type": [
+              "third-party"
+            ]
+          }
+        },
+        "name": "coinhive",
+        "id": "D22F7992-2BEF-4FF6-94CD-63075A872429"
+      },
+      {
+        "content": {
+          "action": {
+            "type": "block"
+          },
+          "trigger": {
+            "url-filter-is-case-sensitive": true,
+            "url-filter": "https?://cnhv\\.co",
```

Comment:
I've never personally heard of 1blocker, so I don't know much about it and I'm presuming this is regex, should this be `\\.`? because that will match a literal `\` and then "any character". See https://regex101.com/r/JjeBX3/1
I may be wrong (I couldn't find any documentation on the syntax for this), but just want to make sure.

Reply:
Just read your comment in the other PR. If it's just an export, don't worry about it. Just the documentation would be good.

Reply:
Yes, it is (possibly a dialect of) regex, but In the GUI in 1Blocker it has a single backslash, It seems the exported file adds a 2nd backslash. It works fine when imported. Most of the documentation for making 1blocker packages seems to exclusively be within it's GUI, it is self explanatory to anyone editing it with 1blocker, but not explained anywhere how to edit the file directly.

Reply:
Sure! Merged 👍